File: test-automation-core/src/main/java/com/nortal/test/core/configuration/IntegrationHostsProperties.java
package com.nortal.test.core.configuration;

import java.util.HashMap;

import com.nortal.test.postman.PostmanHostAware;
import org.springframework.boot.context.properties.ConfigurationProperties;
import org.springframework.boot.context.properties.EnableConfigurationProperties;
import org.springframework.stereotype.Component;
import lombok.Getter;
import lombok.Setter;

@Component
@EnableConfigurationProperties
 @ConfigurationProperties(prefix = "test-automation.integration")
public class IntegrationHostsProperties implements PostmanHostAware {

	@Setter
	@Getter
	private HashMap<String, String> hosts;

}

File: test-automation-core-old/src/main/java/com/nortal/test/core/configuration/IntegrationHostsProperties.kt
package com.nortal.test.core.configuration

import lombok.Getter
import lombok.Setter
import org.springframework.boot.context.properties.ConfigurationProperties
import org.springframework.boot.context.properties.EnableConfigurationProperties
import org.springframework.stereotype.Component
import java.util.HashMap

@Component
@EnableConfigurationProperties
@ConfigurationProperties(prefix = "test-automation.integration")
class IntegrationHostsProperties /*implements PostmanHostAware*/ {
    @Setter
    @Getter
    private val hosts: HashMap<String, String>? = null
}